src/widgets/Updater.cpp

```diff
@@ -641,44 +641,6 @@ bool Updater::Tick(wxDC& dc,
    return !rect.IsEmpty();
 }
 
-bool Updater::TickCustom(wxDC& dc, int labelIdx, wxFont font,
-   // in/out:
-   TickOutputs outputs,
-   const RulerStruct& context) const
-{
```

Comment:
The last commit, too, might as well be squashed onto the commit that introduces Updater.